packages/formstr-app/src/containers/CreateForm/components/InputsMenu/index.tsx

```diff
@@ -1,11 +1,20 @@
+import { useContext } from "react";
 import { Menu } from "antd";
 import { INPUTS_MENU } from "../../configs/menuConfig";
+import { FormBuilderContext } from "../../providers/FormBuilder";
 
 function InputsMenu() {
+  const { addQuestion } = useContext(FormBuilderContext);
```

Comment:
should form builder context expose a `useFormBuilderContext();` ?

Reply:
fixed